develop/parachain-devs/runtime-development/FRAME/create-custom-pallet.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Create a Custom Pallet
+description: Learn how to create custom pallets using FRAME, allowing for flexible, modular, and scalable blockchain development. Follow the step-by-step guide.
+---
+
+# Create a Custom Pallet
+
+## Introduction
+
+To get the most out of this guide, ensure you're familiar with [FRAME concepts](TODO:update-path){target=\_blank}. FRAME provides a powerful set of tools for blockchain development, including a library of pre-built pallets. However, its true strength lies in the ability to create custom pallets tailored to your specific needs. This section will guide you through the process of creating your own custom pallet, allowing you to extend your blockchain's functionality in unique ways.
```

Comment:
IMO, I think starting off the guide with "To get the most out of this guide..." comes off a little strong. I'd recommend wrapping up the initial paragraph with it instead. So users can have some knowledge about what the page is about before getting ushered to another page 🤷‍♀️